app/controllers/abuse_reports_controller.rb

```diff
@@ -1,22 +1,34 @@
 class AbuseReportsController < ApplicationController
-
   skip_before_filter :store_location
+  before_filter :load_abuse_languages
 
   def new
     @abuse_report = AbuseReport.new
-    @abuse_report.url = params[:url] || request.env["HTTP_REFERER"]
-    @abuse_report.email = User.current_user.try(:email)
+    if logged_in_as_admin?
+      @abuse_report.email = current_admin.email
```

Comment:
Should this use the admin login name for the username field as well as filling in the admin's email?